package.json

```
@@ -18,7 +18,9 @@
     "html-webpack-plugin": "^5.5.1",
     "webpack": "^5.79.0",
     "webpack-cli": "^5.0.1",
-    "webpack-dev-server": "^4.13.3"
+    "webpack-dev-server": "^4.13.3",
```

Нужно не забыть зафиксировать версии у всех пакетов

```suggestion
    "webpack-dev-server": "4.13.3",
```